app/policies/food_policy.rb

```diff
@@ -0,0 +1,23 @@
+class FoodPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def create?
+    user.present?
```

Comment:
I am not sure that it is useful now, as Devise controls if you have logged current user or not by `before_action :authenticate_user!`